src/GeneratorV2/Process/Processor.cs

```diff
@@ -527,6 +528,76 @@ public bool TryGenerateOverloads(Overload overload, [NotNullWhen(true)] out List
             }
         }
 
+        class FunctionPtrToDelegateOverloader : IOverloader
+        {
+            public bool TryGenerateOverloads(Overload overload, [NotNullWhen(true)] out List<Overload>? newOverloads)
+            {
+                Parameter[] parameters = new Parameter[overload.InputParameters.Length];
+                List<Parameter> original = new List<Parameter>();
+                List<Parameter> changed = new List<Parameter>();
+                NameTable nameTable = overload.NameTable.New();
```

Comment:
If you do not change thins such as generic types i dont think you should set them explicitly. This would make it a lot harder to go through and change the generic types to other types or remove them. This goes for any properties of the overloads.

This is what we get the ability to do because we are using records. So by explicitly setting everything you might aswell not use the new `with` syntax

Reply:
After some thinking I finally understood that you meant to reference the line below this comment.

Yeah the setting of the generics array was a copy-paste remnant. It's not supposed to be there.